Release.mk

```diff
@@ -0,0 +1,12 @@
+include make-base/utils.mk
```

Comment:
Meta magic time.

`make-base/utils.mk` now includes some super handy make meta functions that are capable of constructing a dependency tree from dockerfiles and creating targets based on that using `.marker` files.

Here, we find every project within monoetna that contains a Dockerfile and instantiate meta targets for its release, testing, and building.  We create one target that runs all of these inner targets as well.  This release process is run in `staging` and `production` branches to release images to dockerhub, which allows our production dockers to pick them up and udpate their images.